lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -187,15 +194,17 @@ def _request_certificate(
             **self._get_sans(cert_type),
         )
 
+        cert_data = {
+            "key": key.decode("utf-8"),
+            "key-password": password,
+            "csr": csr.decode("utf-8"),
+            "subject": f"/O={self.charm.app.name}/CN={subject}",
+        }
```

Comment:
```suggestion
```

Reply:
Bah, I think that was a debug thing, removeing.